packages/core/src/middleware/koa-security-headers.ts

```diff
@@ -0,0 +1,49 @@
+import type { MiddlewareType } from 'koa';
+import compose from 'koa-compose';
+import helmet from 'koa-helmet';
+
+export default function koaSecurityHeaders<StateT, ContextT, ResponseBodyT>(): MiddlewareType<
+  StateT,
+  ContextT,
+  ResponseBodyT
+> {
+  return compose([
+    // Expect-ct: https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Headers_Cheat_Sheet.html#expect-ct  not recommended
+    // helmet.expectCt(),
+
+    // X-Powered-By: https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Headers_Cheat_Sheet.html#x-powered-by
+    helmet.hidePoweredBy(),
```

Comment:
how about just `helmet()` to avoid such long middleware chain

Reply:
Intend to avoid applying some helmet default headers.